### DISCUSSION TITLE
(Vue) Date preview not showing in qdate element

### DISCUSSION THREAD

**Discussion by abnormalyak** [quasarframework/quasar#10442] [General - Components / Directives / etc] [answered]
Hi there, when I use the following mask... 
`mask="dddd Do of MMM h:mmA"`
... Then, although the actual data is formatted as I desired (e.g. "Friday 20th of November, 5:00PM"), the actual qdate element's preview is empty: 
![image](https://user-images.githubusercontent.com/41366885/130253489-42978290-1feb-406e-ac73-d52024491a55.png)
Clicking on dates will save that value, but it isn't shown to the user.

How can I fix this / what is causing this? 

Thank you for any help.

The code: https://codeshare.io/9OXxN7

**Comment by pdanpdan** [comment]
You have a missing coma in the mask, after the month

**Reply by abnormalyak** [reply]
Thank you for the response, I've just tried adding a comma after the month, however the issue still remains - do you know what could be causing it? It is functional, but the selection part of the qdate element is empty and there isn't a circle around the date I select.

**Reply by ontwikkelfabriek** [reply]
@abnormalyak 
Create a codepen.io and we can tell much better what is wrong.

**Comment by pdanpdan** [comment, marked as answer]
You don't have the year

**Reply by abnormalyak** [reply]
Thank you! That worked. I did not realize it was compulsory, I will trim off the year in the final result.
